### PR TITLE
separate read and write aliases 

### PR DESCRIPTION
**Pull request by NickPapacostas**
used in media-api and thrall

### CODE REVIEW

File: thrall/app/lib/Config.scala [hunk 1]
```diff
@@ -23,7 +23,7 @@ object Config extends CommonPlayAppConfig {
 
   val imageBucket: String = properties("s3.image.bucket")
 
-  val imagesAlias = properties("es.index.alias")
+  val writeAlias = properties("es.index.aliases.write")
```
Comment (akash1810) [inline thrall/app/lib/Config.scala]: are we sure thrall doesn't need the read alias too?

File: scripts/src/main/scala/com/gu/mediaservice/scripts/EsScript.scala [hunk 2]
```diff
@@ -13,7 +13,7 @@ object MoveIndex extends EsScript {
   def run(esHost: String, extraArgs: List[String]) {
 
     object EsClient extends ElasticSearchClient {
-      val imagesAlias = "imagesAlias"
+      val imagesAlias = "readAlias"
```
Comment (akash1810) [inline scripts/src/main/scala/com/gu/mediaservice/scripts/EsScript.scala]: can this be read from the `Config` object like [here](https://github.com/guardian/grid/pull/1701/files#diff-c8a14be53a6e8680f2e620d6c17edaadR56)

Comment (NickPapacostas) [inline scripts/src/main/scala/com/gu/mediaservice/scripts/EsScript.scala]: I'm not sure tbh. I couldn't get it workings and there was this comment further down in the file: 
  // FIXME: Get from config (no can do as Config is coupled to Play)

File: thrall/app/lib/Config.scala (hunk 1 above)
Comment (NickPapacostas) [inline thrall/app/lib/Config.scala]: I actually don't think thrall _should_ touch the read alias. in the sense that, if it's query-ing to determine what to do next, it should be doing so on the write alias shouldn't it? (<- not rhetorical ^_^) 

I double checked anyway and I don't see anywhere in Thrall where a query reads w/o writing.

**Comment by akash1810** [comment]
:+1:

**Pull request merged**